egs2/msuperb/asr1/run_single_lang.sh

```diff
@@ -39,6 +39,7 @@ for duration in 10min 1h; do
             --feats_type raw \
             --asr_config "${asr_config}" \
             --inference_config "${inference_config}" \
+            --inference_asr_model "valid.loss.ave_5best.pth" \
```

Comment:
I think the default one is the ave ones right? In that case, no need to change it

Reply:
oh yes I forgot about this softlink